RecoEgamma/EgammaHLTProducers/plugins/HLTCaloObjInRegionsProducer.cc

```diff
@@ -326,8 +310,6 @@ DEFINE_FWK_MODULE(HLTHcalDigisInRegionsProducer);
 using HLTHcalQIE11DigisInRegionsProducer = HLTCaloObjInRegionsProducer<QIE11DataFrame, QIE11DigiCollection>;
 DEFINE_FWK_MODULE(HLTHcalQIE11DigisInRegionsProducer);
 
-#include "DataFormats/HcalDigi/interface/QIE10DataFrame.h"
```

Comment:
This one is still needed (above it is included QIE11DataFrame.h instead)

Reply:
Thanks, that was really easy to miss!